cmd/kubeadm/app/cmd/upgrade/node.go

```diff
@@ -22,14 +22,21 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/golang/glog"
+	"github.com/pkg/errors"
```

Comment:
I've mostly seen fmt.Errorf instead of using errors.Wrap in the kubeadm code base. We should be consistent. Let me know if I missed the decision to switch over to `pkg/errors`.

Reply:
I'm using errors.Wrap because I was asked to in [66873](https://github.com/kubernetes/kubernetes/pull/66873#discussion_r208609644). I agree we should be consistent - an possibly have a sort of kubeadm coding guidelines somewhere -, so let's raise the point at the next kubeadm office hour

Reply:
Heptio uses errors.Wrap for our stuff, but kubeadm has always been fmt.Errrorf. I think @timothysc got confused. That comment provided no reasons for the change so let's stick with fmt.Errorf. It's easy enough to fix later if there we decide to move that direction, otherwise I don't want to introduce inconsistency here.